src/tests/receiving/importReceivingTemplate.test.ts

```diff
@@ -0,0 +1,215 @@
+      (row[11] = '20'), (row[12] = 'e2e-comment');
+      data.push(row);
+    });
+
+    const fileName = 'modified.csv';
+    const fullFilePath = path.join(AppConfig.LOCAL_FILES_DIR_PATH, fileName);
+
+    await test.step('Save file', async () => {
+      const savedFile = WorkbookUtils.saveFile(data, fullFilePath);
+      workbooks.push(savedFile);
+    });
+
+    await test.step('Upload edited file', async () => {
+      await receivingPage.receivingStep.uploadFile(fullFilePath);
+    });
+
+    await test.step('Assert inupt values after import file', async () => {
+      await receivingPage.receivingStep.isLoaded();
+      await expect(
+        receivingPage.receivingStep.table.row(1).receivingNowField.textbox
+      ).toHaveValue('20');
+      await expect(
+        receivingPage.receivingStep.table.row(1).commentField.textbox
+      ).toHaveValue('e2e-comment');
+    });
+
+    await test.step('Go to Check page', async () => {
+      await receivingPage.receivingStep.isLoaded();
+      await receivingPage.nextButton.click();
+    });
+
+    await test.step('Receive shipment', async () => {
+      await receivingPage.checkStep.isLoaded();
+      await receivingPage.checkStep.receiveShipmentButton.click();
+      await stockMovementShowPage.isLoaded();
+      await expect(stockMovementShowPage.statusTag).toHaveText('Received');
+    });
+  });
+
+  test('Display validation when try to edit other files through import', async ({
+    stockMovementShowPage,
+    receivingPage,
+  }) => {
+    let filePath: string;
+    let downloadedExportTemplateFile: WorkbookUtils;
+
+    await test.step('Go to stock movement show page', async () => {
+      await stockMovementShowPage.goToPage(STOCK_MOVEMENT.id);
+      await stockMovementShowPage.isLoaded();
+    });
+
+    await test.step('Go to receiving page', async () => {
+      await stockMovementShowPage.receiveButton.click();
+      await receivingPage.receivingStep.isLoaded();
+    });
+
+    await test.step('Download export template', async () => {
+      const { fullFilePath } =
+        await receivingPage.receivingStep.downloadExportTemplate();
+      filePath = fullFilePath;
+    });
+
+    let parsedDocumentData: unknown[][];
+
+    await test.step('Read downloaded file', async () => {
+      downloadedExportTemplateFile = WorkbookUtils.read(filePath);
+      workbooks.push(downloadedExportTemplateFile);
+    });
+
+    await test.step('Parse csv document to json', async () => {
+      parsedDocumentData = downloadedExportTemplateFile.sheetToJSON();
+    });
+
+    const data: unknown[][] = [];
+
+    await test.step('Parse downloaded template document', async () => {
+      data.push(downloadedExportTemplateFile.getHeaders());
+    });
+
+    await test.step('Input receiving now qty into template', async () => {
+      const documentRow = parsedDocumentData[1];
+      const row = [...documentRow];
+      (row[4] = 'editlot'), (row[11] = '20'), (row[12] = 'e2e-comment');
```

Comment:
Can we add consts holding these column numbers that would describe what is behind it?